.github/stale.yml

```diff
@@ -1,25 +1,34 @@
 # Number of days of inactivity before an issue becomes stale
 daysUntilStale: 30
+
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 14
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels:
+  - "additional information requested"
```

Comment:
Feels long for a tag. How about something like `needs more info` or `more info required`?